### ISSUE TITLE
Black screen flash when opening rom

### ISSUE THREAD

**Issue by srose24** [zach-morris/plugin.program.iarl#41]
Hello,

I just followed the instructions (I'm pretty sure correctly) to get this setup on my kodi/firestick.  Everything appears to work properly, but when I launch a rom, everything downloads ok and then the screen flashes black and then goes back to the rom launcher screen.  Any idea why this is happening?  Any help would be greatly appreciated.

Thanks!
Scott

**Comment by zach-morris** [comment]
Without logs there's no way to know.  I suggest following [these troubleshooting steps](https://github.com/zach-morris/plugin.program.iarl/wiki/5.--FAQ#14b-i-pressed-launch-but-nothing-happened-your-addon-is-broken).

**Comment by srose24** [comment]
Thanks for getting back to me and sorry about no including the log files.  Here is the link: https://paste.ubuntu.com/25173123/

I looked at the troubleshooting steps and went through the setup process you outlined again to make sure I didn't miss anything.  I obviously have somewere, But I just don't know where...  I would appreciate it if you could look over the log for me and see if anything sticks out to you that may be causing the issue.

Thanks again for your time!

Go Bayside High!!  lol

Scott

**Comment by zach-morris** [comment]
The game you try to launch is a Coleco game.  IARL looks like it downloads it correctly and then sends the command to try and launch the game with bluemsx.  Here's the command the addon sends:
`/system/bin/am start --user 0 -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -e ROM "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/userdata/addon_data/plugin.program.iarl/temp_iarl/coleco/ladybug.zip" -e LIBRETRO /data/data/com.retroarch/cores/bluemsx_libretro_android.so -e CONFIGFILE /storage/emulated/0/Android/data/com.retroarch/files/retroarch.cfg -e IME com.android.inputmethod.latin/.LatinIME -n com.retroarch/.browser.retroactivity.RetroActivityFuture`


At any rate, for this particular system the issue is that the rom file isn't compatible with BlueMSX without probably a lot of finagling.  The romset used currently for Coleco is for MAME/MESS, and the [default core](https://github.com/zach-morris/plugin.program.iarl/blob/master/resources/lib/run_iarl_wizard.py#L756) IARL tries to use with this game list is MAME.

Suggest trying to download the core MAME and/or MESS2014 in Retroarch, then update the external launch command to MAME or MESS2014 and try it again.

If you get the same thing, then try launching the game directly in the Retroarch app by browsing to the file that IARL downloads and launching it.

**Comment by srose24** [comment]
Gotcha on the BlueMSX core.  I wasn't sure and then googled what I thought I needed to google and saw a reference to the BlueMSX.  After changing it to MAME, I still get a delayed black screen then it goes back to the rom launcher screen.  When I try to manually load it from Retroarch, it doesn't load from there either...  I've been able to get other roms to work, like in NES and Atari, but I'm really hoping to get the Colecovision roms to work as I had one of those growing up and there are several games that I would love to be able to play again on this...  Not sure if you have any other ideas on this or not, but if so, I would appreciate your suggestions.

Here is the latest log if it helps...

https://paste.ubuntu.com/25181058/

Thanks.
